nautobot/ui/src/utils/store.js

```diff
@@ -10,45 +12,131 @@ import {
     REGISTER,
 } from "redux-persist";
 import storage from "redux-persist/lib/storage";
-import { combineReducers } from "redux";
-import { createSlice } from "@reduxjs/toolkit";
 
-// Configure redux-persist to save state in Local Storage for performance
-const persistConfig = {
-    key: "root",
-    version: 1,
-    storage,
+//      ___         __  __
+//    /   | __  __/ /_/ /_
+//   / /| |/ / / / __/ __ \
+//  / ___ / /_/ / /_/ / / /
+// /_/  |_\__,_/\__/_/ /_/
```

Comment:
Pretty as ASCII-art can be it's not very useful from a searchability/discoverability standpoint. I'd prefer something simple like just:

```suggestion
//
// AUTH
//
```